bliss/conf/base_config.yaml

```diff
@@ -114,7 +115,7 @@ encoder:
 generate:
     n_workers_per_process: 0
     n_batches: ${simulator.n_batches}
-    batch_size: ${simulator.prior.batch_size}
+    batch_size: ${simulator.survey.prior_config.batch_size}
```

Comment:
The survey shouldn't know about the batch size. Batch size governs how we generate data, it's not a property of the survey.

Reply:
Done.